website/content/en/docs/deployment/create-eks-cluster.md

```diff
@@ -35,6 +35,10 @@ eksctl create cluster \
 --with-oidc
 ```
 
+{{% alert title="Note" color="info" %}}
+Kubeflow only supports x86-based EC2 worker nodes at this time. Efforts are underway to provide Kubeflow support for AWS Graviton instances as well.
```

Comment:
Efforts are underway to provide Kubeflow support for AWS Graviton instances as well.

Either remove this or I would recommend linking to an existing GH issue on OSS to follow